pubtools/_quay/container_image_pusher.py

```diff
@@ -128,11 +128,16 @@ def _prepare_dest_refs(self, push_item):
         for repo, tags in sorted(push_item.metadata["tags"].items()):
             internal_repo = get_internal_container_repo_name(repo)
             for tag in tags:
+                dst_tag = (
+                    f"{tag}.{push_item.metadata.get('com.redhat.prerelease')}"
```

Comment:
The tag created provided by ET will have a 'com.redhat.prerelease' label value already, if we add it here again, the  final tag would have the label value twice, can't we use the tag from ET directly?

Reply:
Yeah, I forgot they have tag template for prerelease already there